packages/twofish/build.sh

```
@@ -0,0 +1,31 @@
+  -fno-builtin \
+  -I. \
+  -Isrc/c \
+  -Wl,--no-entry \
+  -Wl,--export=twofish_init \
+  -Wl,--export=twofish_get_io_buffer \
+  -Wl,--export=twofish_create_session \
+  -Wl,--export=twofish_encrypt \
+  -Wl,--export=twofish_decrypt \
+  -Wl,--export=twofish_destroy_session \
+  -o dist/twofish.wasm \
+  $(find src -name '*.c')
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)